python3/agents/shared/strategies/stalk_strategy.py

```diff
@@ -27,3 +27,26 @@ def execute(self, game_state: object) -> List[str]:
             return [action_seq.pop(0)]
         else:
             return action_seq
+
+
+class StalkTwoStrategy(strategy.Strategy):
+
+    def execute(self, game_state: object) -> List[str]:
+        enemy_pos = game_state['enemy_pos']
+        player_pos = game_state['player_pos']
+        world = game_state['world']
+        entities = game_state['entities']
+
+        # we want to at least be 3 blocks away from player
+        closest_tile =  game_state['closest_bomb_tile']
+
+        path = get_shortest_path(player_pos, closest_tile, world, entities, game_state['hazard_zones'], game_state['player_is_invulnerable'])
+        # if path:
```

Comment:
how come this is removed?

Reply:
since we're popping the list later but removing this first, it never actually wants to go orthogonal to an enemy (I want it to go orthogonal to an enemy so it can trap/place bombs better)